src/openzaak/utils/cache.py

```diff
@@ -110,13 +111,28 @@ def requests_cache_enabled(cache_name, backend=None, *args, **kwargs):
 
     original_client = vng_api_common.client.Client
 
+    cache_name = "import_requests"
```

Comment:
There are two options to set up cache backend for requests_cache now:
* `'import_requests'` constant (like here and in CACHE setting
* `IMPORT_REQUESTS_CACHE_NAME` setting like default in DjangoRequestsCache 

It seems possible that if IMPORT_REQUESTS_CACHE_NAME is configured the default requests-cache backend could appear again. Anyway I think IMPORT_REQUESTS_CACHE_NAME can be removed to reduce this probability

Reply:
Hmm yeah I think I might just make it always use the same cache name + backend without making it configurable, to avoid this issue